assets/blocks/course-outline/outline-block/elements/ai-lessons-image.js

```diff
@@ -0,0 +1,26 @@
+const AiLessonsImage = () => (
+	<div className="ai-course-outline-image">
+		<div className="ai-course-outline-image__item">
+			<div
+				style={ { width: '78%' } }
```

Comment:
Curious why we're using inline styles here?

Reply:
I understood it is just a static image implemented using HTML to be "fluid"  (I haven't found an easy way to do it with SVG) and considering it is just a block interface that never will be rendered outside the editor, I think it should not be influenced by the theme.

Considering it and the fact I got myself adding very specific classes to do the exact thing like`w78`  I couldn't see a reason to implement it via inline attribute.

What do you think?